docs/docker.md

```diff
@@ -2,43 +2,132 @@
 
 There is now an official [uMap](https://hub.docker.com/r/umap/umap) image.
 
-To run it with docker compose, use a `docker-compose.yml` like this:
+To run it with docker compose, create a `docker-compose.yml` like this:
 
+_Sample docker-compose.yml_ 
 ```yaml
 version: '3'
 services:
   db:
-    image: postgis/postgis:14-3.3-alpine
+    # check https://hub.docker.com/r/postgis/postgis to see available versions
+    image: postgis/postgis:14-3.4-alpine
     environment:
       - POSTGRES_HOST_AUTH_METHOD=trust
     volumes:
-      - db:/var/lib/postgresql/data
+      - umap_db:/var/lib/postgresql/data
 
   app:
-    image: umap/umap:x.x.x
+    # Check https://hub.docker.com/r/umap/umap/tags to find the latest version
+    image: umap/umap:1.3.7
     ports:
+      # modify the external port (8001, on the left) if desired, but make sure it matches SITE_URL, below
       - "8001:8000"
     environment:
       - DATABASE_URL=postgis://postgres@db/postgres
-      - SECRET_KEY=some-long-and-weirdly-unrandom-secret-key
-      - SITE_URL=https://umap.local/
-      - UMAP_ALLOW_ANONYMOUS=True
-      - DEBUG=1
+      - SITE_URL=https://localhost:8001/
+      - STATIC_ROOT=/srv/umap/static
+      - MEDIA_ROOT=/srv/umap/uploads
     volumes:
-      - data:/srv/umap/uploads
-
+      - umap_userdata:/srv/umap/uploads
+      # FIX the path on the left, below, to your location 
+      - /home/ubuntu/umap.conf:/etc/umap/umap.conf
+    restart: always
+    depends_on:
+      - db
+    
 volumes:
-  data:
-  db:
+  umap_userdata:
+  umap_db:
+
 ```
 
-### Create superuser
+Next, create a basic settings file, named `umap.conf` in the same directory. 
+
+You can use this example below and it will run, but you may want to look at the project sample config, using `wget https://raw.githubusercontent.com/umap-project/umap/master/umap/settings/local.py.sample -O /etc/umap/umap.conf` and modify as needed. 
+
+Make sure the settings in the docker-compose don't conflict with the sample config and vice-versa. In particular, remove the DATABASES section from the config file if useing the docker-compose file, or it will override the DATABASE_URL setting and things won't work. 
```

Comment:
```suggestion
Make sure the settings in the docker-compose don't conflict with the sample config and vice-versa. In particular, remove the DATABASES section from the config file if using the docker-compose file, or it will override the DATABASE_URL setting and things won't work. 
```